test/views/authenticated/ChangeWhichTaxYearsViewSpec.scala

```diff
@@ -49,20 +49,13 @@ class ChangeWhichTaxYearsViewSpec extends CheckboxViewBehaviours[TaxYearSelectio
 
     behave like pageWithBackLink(applyView(form))
 
-    "contains correct headings for table" in {
-      val doc = asDocument(applyView(form))
-
-      doc.getElementById("tax-year-heading").text mustBe messages(s"$messageKeyPrefix.columnHeading1")
-      doc.getElementById("amount-heading").text mustBe messages(s"$messageKeyPrefix.columnHeading2")
-    }
-
-    "contains correct column values for table" in {
+    "contains correct values for list" in {
       val doc = asDocument(applyView(form))
 
       taxYearsAndAmounts.map{
         options =>
-          doc.getElementById(s"tax-year-${options._1.value}").text mustBe s"${options._1.message.html}"
-          doc.getElementById(s"fre-amount-${options._1.value}").text mustBe s"£${options._2}"
+          doc.getElementById(s"tax-year-${options._1.value}").text contains s"${options._1.message.html}"
+          doc.getElementById(s"fre-amount-${options._1.value}").text contains s"£${options._2}"
```

Comment:
This could be done with: 
`doc.getElementById(s"fre-amount-${options._1.value}").text mustBe s"${messages("changeWhichTaxYears.columnHeading2")} £${options._2}"`